README.md

```diff
@@ -1,28 +0,0 @@
-# Elasticsearch deployment on EKS K8s
```

Comment:
Why was this file deleted?

Reply:
MB, deleted the wrong file. Will revert in new commit.